docs/sources/developer/Developers-Guide.md

```diff
@@ -66,7 +66,7 @@ If your code requires adding a new dependency please check out these instruction
 
 ### Before you submit your first code review
 
-1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/?fromgroups#!forum/log2timeline-dev)
+1. Join the development mailing list: [log2timeline-dev@googlegroups.com](https://groups.google.com/forum/#!forum/log2timeline-dev)
```

Comment:
In docs/sources/user/Overview.md, the '!' character is changed into %21, not sure whether there is a need to change it here or there

Reply:
thx for flagging, I'll change it here